core/src/main/java/io/kestra/core/services/TaskDefaultService.java

```diff
@@ -70,14 +72,15 @@ public Flow injectDefaults(Flow flow, Execution execution) {
     public Flow injectDefaults(Flow flow, Logger logger) {
         try {
             return this.injectDefaults(flow);
-        } catch (Exception e) {
+        } catch (ConstraintViolationException e) {
```

Comment:
Why change from Exception to ConstraintViolationException? 
It's a security that need to be kepts to avoid any exception that could crash the executor